app/src/main/java/org/oppia/app/databinding/TextViewBindingAdapters.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.app.databinding
+
+import android.text.format.DateUtils
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import java.util.*
+
+/** Converts time in ms to readable relative time string. */
+@BindingAdapter("profile:date")
+fun setTextWithDate(textView: TextView, timeMs: Long) {
+  val dateText = "Last used " + DateUtils.getRelativeTimeSpanString(timeMs, Date().time, DateUtils.MINUTE_IN_MILLIS, DateUtils.FORMAT_ABBREV_RELATIVE)
```

Comment:
We should never show text to the user that's manually concatenated. Instead, we should use string interpolation with a strings.xml string so that translators can move around the sentence structure to match different locales.

Reply:
Removed file.